R-TypeGame/Game/GameData/Scenes/firstScene.json

```diff
@@ -88,14 +88,6 @@
             "path": "libupdate_playerConnection_system.so"
         }
     ],
-    "menus": {
-        "keyBinds": [
-            {
-                "key": "KEY_F",
-                "path" : "Menus/mainMenu.json"
-            }
-        ]
-    },
```

Comment:
Don't delete the menus field, just the object path and key.

Reply:
Ok no problems